go.mod

```diff
@@ -187,6 +189,7 @@ require (
 	github.com/apache/thrift v0.18.1 // indirect
 	github.com/armon/circbuf v0.0.0-20190214190532-5111143e8da2 // indirect
 	github.com/armon/go-radix v1.0.0 // indirect
+	github.com/aws/aws-sdk-go v1.55.6 // indirect
```

Comment:
Thanks for the contribution! We will take a look as soon as we can.

One small thing: Do you know which dependency requires AWS-SDK-Go v1?
We recently (to be released in 4.71.0 any time soon) did a big migration from v1 to v2 for security purposes (see https://github.com/fleetdm/fleet/issues/29482).

Reply:
Good catch, fixed by pulling in a newer version of `rdsmysql`